Content.Shared/Item/ItemToggle/Components/ItemToggleComponent.cs

```diff
@@ -19,6 +19,12 @@ public sealed partial class ItemToggleComponent : Component
     [DataField, AutoNetworkedField]
     public bool Activated = false;
 
+    /// <summary>
+    /// Can the entity be toggled with a verb or interaction.
+    /// </summary>
+    [DataField]
+    public bool ManualToggle {get; set;} = true;
+
```

Comment:
This just doubles onactivate?

Reply:
If that is the case Ill use that instead